examples/i3s/deck.gl/app.js

```diff
@@ -47,16 +49,35 @@ export default class App extends PureComponent {
     const parsedUrl = new URL(window.location.href);
     const renderCesium = parsedUrl.searchParams.get('cesium');
     this.setState({renderCesium});
+
+    this._memWidget = new StatsWidget(lumaStats.get('Memory Usage'), {
+      framesPerUpdate: 1,
+      formatters: {
```

Comment:
Any chance of getting the default formatters registered in luma.gl? This verbose code is now in more and more examples...

This is a great timing, now that luma.gl is making a major release version so no fear of accidentally breaking apps.